### ISSUE TITLE
Regarding Test Data and Validation Data

### ISSUE THREAD

**Issue by RRRRendy** [egyptdj/stagin#19]
It seems that the validation process and the testing process have used the same data partitioning method and input the same data. I'm not quite clear if I'm seeing this wrong, but I think the validation set should be divided from the training set, right?

**Comment by RRRRendy** [comment]
if argv.validate: ....
Is it unused, right?

**Comment by yusufm423** [comment]
Same issue!
